<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTournamentSettings</class>
 <widget class="QDialog" name="DlgTournamentSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit tournament settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tournament name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTournamentName"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Organizing club:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leOrgaClub"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Default umpire mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbUmpire"/>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
       <item>
        <widget class="QLabel" name="laCourtCount">
         <property name="text">
          <string>Number of courts:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbCourtCount">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOkay">
         <property name="text">
          <string>Okay</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgTournamentSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOkay</sender>
   <signal>clicked()</signal>
   <receiver>DlgTournamentSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leOrgaClub</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DlgTournamentSettings</receiver>
   <slot>onOrgaNameChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTournamentName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DlgTournamentSettings</receiver>
   <slot>onTournamentNameChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbUmpire</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DlgTournamentSettings</receiver>
   <slot>onUmpireSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTournamentNameChanged()</slot>
  <slot>onOrgaNameChanged()</slot>
  <slot>onUmpireSelectionChanged()</slot>
 </slots>
</ui>
